feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewScreen.kt

```diff
@@ -104,11 +111,13 @@ fun PreviewScreen(
             onCaptureImageWithUri = viewModel::captureImageWithUri,
             onStartVideoRecording = viewModel::startVideoRecording,
             onStopVideoRecording = viewModel::stopVideoRecording,
-            onToastShown = viewModel::onToastShown
+            onToastShown = viewModel::onToastShown,
+            onSnackBarResult = viewModel::onSnackBarResult
         )
     }
 }
 
+@SuppressLint("UnusedMaterial3ScaffoldPaddingParameter")
```

Comment:
Are we sure we aren't supposed to be using the padding parameter here?

Reply:
"Scaffold padding values define padding that will be applied along the inside edges of a box.

The value will only be provided if the scaffold view’s bottomBar is set. Otherwise, it will only return 0.0 dp."

Since we don't have a bottomBar here, I think we have no use for the padding value